base/src/main/java/butter/droid/base/manager/internal/vlc/VlcPlayer.java

```diff
@@ -55,8 +55,10 @@ public boolean initialize() {
             return false;
         }
 
-        mediaPlayer = new MediaPlayer(libVLC);
-        mediaPlayer.setEventListener(this);
+        if (mediaPlayer == null) {
```

Comment:
if we call twice `initialize` we may have a new `mediaPlayer`. would be better to destroy the previous one?

Reply:
it would probably be the beast to just throw exception saying that it was already initialized.

Reply:
same for the line above so. and what do we do with the `boolean` return? I am agree with you but I would make it `void` at this point